integreat_cms/release_notes/current/unreleased/3279.yml

```
@@ -0,0 +1,2 @@
+en: Export only selected page access statistics
```

```suggestion
en: Export only statistics for selected languages
```
Since this is mostly about the language selection, I find this to be a bit more speaking, but I might be wrong as well. Also I think "page access statistics" might be a bit confusing, since to me it carries the same meaning as "seitenbasierte Zugriffe", and those are not yet implemented